source/globalCommands.py

```diff
@@ -1786,7 +1785,7 @@ def script_review_copy(self, gesture):
 				pass
 			try:
 				copyMarker.updateSelection()
-				if isinstance(pos.obj, editableText.EditableTextWithoutAutoSelectDetection):
+				if hasattr(pos.obj, "waitForAndReportSelectionChange"):

```

Comment:
Are there other classes that may need to implement this? What other type can `obj` be?

Rather than implement this on `cursorManager` (which does not need to wait) perhaps a better "catch-all" would be to put the newly added contents of `cursorManager.waitForAndReportSelectionChange` into the `else` block for this condition. That would also get around the naming problem.